### ISSUE TITLE
Training on non-model organisms

### ISSUE THREAD

**Issue by kyleoconnell** [google/deepvariant#459]
**Have you checked the FAQ? https://github.com/google/deepvariant/blob/r1.1/docs/FAQ.md**:

**Describe the issue:**
(A clear and concise description of what the issue is.)

**Setup**
  - Operating system:
  - DeepVariant version:
  - Installation method (Docker, built from source, etc.):
  - Type of data: (sequencing instrument, reference genome, anything special that is unlike the case studies?)

**Steps to reproduce:**
  - Command:
  - Error trace: (if applicable)

**Does the quick start test work on your system?**
Please test with https://github.com/google/deepvariant/blob/r0.10/docs/deepvariant-quick-start.md.
Is there any way to reproduce the issue by using the quick start?

**Any additional context:**

**Comment by kyleoconnell** [comment]
Hi, I am interested in benchmarking DV on exome data from a group of snakes. I have a single reference genome from a closely related species, but I would not have the trio data as was done with the Drosophila data. Do you know if anyone has been able to retrain DV using one or two genomes without pedigree info? 

Thanks.

**Comment by danielecook** [comment]
Hi @kyleoconnell in order to train a model you will need to establish ground truth data. This can be used both for training and evaluation purposes.

Previously, DeepVariant has been applied to mouse data with a human model and performed well. However, even here you will want some basis upon which to determine the quality of the genotypes.

**Comment by kyleaoconnell22** [comment]
Thanks Daniel, non sure if Google Health has tried this at all with non-model genomes, but what I am thinking is trying to train my model based on GATK calls from a deeply-sequenced individual and then evaluate using the same strategy on a second deeply-sequenced individual. I don't have triads so I can't use the method GH used for mosquitoes, but does that sound like a reasonable approach to getting DV off the ground without gold-truth data?

**Comment by AndrewCarroll** [comment]
Hi @kyleaoconnell22 

Can I ask a few other questions - first, have you already attempted to use the human model, and, if so, do you have any indication of issues?

Second, do you know some of the rough properties of the genome (does it have a high repeat content? Do you know the approximate variant density and heterozygosity)?

We have been doing some experimentation with silver standard training data. We don't have any conclusive recommendations. We have thought about ising GATK for the silver lablels, but we're worried that this might carry the sort of artifacts that GATK makes into the deep learning model. Another idea we are looking at is to subset the Genome in a Bottle labels to regions which are more similar to the properties of the species to train a model for. I would suggest that this might be more promising as an approach.

**Comment by kyleaoconnell22** [comment]
Hi @AndrewCarroll , 

Thanks for the detailed reply. This is still kind of in early days. Our team is mostly from the National Museum of Natural History so we all work on a variety of taxa from snakes to plants. At this point we were trying to figure out the best non-model group(s) to start with because if we could start with a taxon with genome trios sequenced (or I suppose WES) then we could try the technique from the mosquito paper. But, in most cases with non-model groups we don't have any trios, and often we just have one representative genome that at times is not that well resolved. In terms of choosing test groups we could try one with genome characteristics more like humans, and then test of something more different. 
 I certainly like the second suggestion of sampling the GIAB data, and if the genome in question is not too different then it could be a good approach. One thing we are thinking about is how far out evolutionarily you can extend the model we train. In a lot of pop gen and phylogenetic studies sampling is interspecific, and so we are also thinking about trying to evaluate how well a model would work across a clade or genus. The fact that DV worked well on mice is encouraging in this respect. I will keep you posted if we come up with any breakthroughs!

**Comment by pichuan** [comment]
Hi @kyleaoconnell22 , I'm closing this issue now. If you have any more questions or results to share, please feel free to follow up!
